analysisData/dog.css

```diff
@@ -0,0 +1,7 @@
+h1 {
```

Comment:
It would be better to use existing `testData` folder and read files from there for test.